Auto Birthday Wisher/README.md

```diff
@@ -0,0 +1,47 @@
+# Auto Birthday Wisher
+
+One forgets to send birthday wishes to friends many times. At such times an automatic birthday wisher comes handy. An automatic birthday wisher via email makes one's life easy. It will send the birthday wishes to friends via email automatically via a server and using an excel sheet to store the data of friends and their birthdays along with email id. It'll send the wishes to friends for all the upcoming years untill we stop the server.
+
+## Setup instructions
+
+In order to run this script, You just need the following modules:
+
+- **Pandas** is a fast, powerful, flexible and easy to use open source data analysis and manipulation tool,
+built on top of the Python programming language.
+```bash
+pip install pandas
+```
+
+- **OpenPyxl** is a Python library to read/write Excel 2010 xlsx/xlsm/xltx/xltm files.

```

Comment:
This library is not used in the script. You can remove this

Reply:
Done with all the changes @kaustubhgupta. You can review them now. 
Thanks!